src/OpenTelemetry.Exporter.InMemory/InMemoryExporterOptions.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="InMemoryExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter
+{
+    public class InMemoryExporterOptions
+    {
+        /// <summary>
+        /// Gets or sets the metric export interval.
+        /// </summary>
+        public int MetricExportInterval { get; set; } = 1000;
```

Comment:
not this PR, but we'll need to make the units obvious for the time interval! (similar to tracing. eg: ScheduledDelayMilliseconds)